spec/double_entry/reporting/line_aggregate_filter_spec.rb

```diff
@@ -78,8 +85,31 @@
       end
     end
 
-    context 'with no code specified' do
+    context 'with a partner_account specified' do
```

Comment:
Some suggestions
1) Do we need `let(:code) { nil }` to make it clear? 
2) Should it be `with a partner_account specified and code not specified` or may a sub-context?
3) Do we need a test for `with code specified and partner_account not specified`?